src/tiles/grid/account-tile.ts

```diff
@@ -2,22 +2,140 @@ import { css, html, LitElement } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
 import { TileModel } from '../../models';
 
+import accountIcon from './icons/account';
+import favoriteFilledIcon from './icons/favorite-filled';
+import reviewsIcon from './icons/reviews';
+import uploadIcon from './icons/upload';
+
 @customElement('account-tile')
 export class UserTile extends LitElement {
   @property({ type: Object }) model?: TileModel;
 
   render() {
     return html`
-      <h1>User</h1>
-      <h2>${this.model?.title}</h2>
+      <div class="outter-holder">
+        <div class="inner-holder">
+          <div class="title-holder">
+            <h1>${this.model?.identifier}</h1>
+          </div>
+          <div class="avatar-holder">
+            <div
+              class="avatar"
+              style="background: url('https://archive.org/services/img/${this
+                .model?.identifier}'); background-position: center;"
+            ></div>
+          </div>
+          <div class="credit-holder">
+            <div class="archivist-since">
+              <h3>Archivist Since</h3>
+            </div>
+            <div class="year-holder">
+              <h3>${this.model?.addeddate?.getFullYear()}</h3>
+            </div>
+          </div>
+          <div class="status-holder">
+            <div class="patron">${accountIcon}</div>
+            <div class="stat-icons">
+              ${uploadIcon}
+              <h3>${this.model?.itemCount}</h3>
+            </div>
+            <div class="stat-icons">
+              ${favoriteFilledIcon}
+              <h3>${this.model?.favCount}</h3>
+            </div>
+            <div class="stat-icons">
+              ${reviewsIcon}
+              <h3>${this.model?.commentCount}</h3>
+            </div>
+          </div>
+        </div>
+      </div>
     `;
   }
 
   static get styles() {
     return css`
       h1 {
-        color: green;
-        margin-top: 0;
+        color: black;
+        font-size: 16px;
+        margin: 0;
+      }
+
+      h2 {
+        margin: 0;
+      }
+
+      h3 {
+        font-size: 14px;
+        font-weight: bold;
+        color: #2c2c2c;
+        margin: 0px;
+      }
+
+      .outter-holder {
+        background-color: #fcf5e6;
+        border: 1px #2c2c2c;
+        border-radius: 4px;
+        box-shadow: 1 1 2 0;
+        height: 100%;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        text-align: center;
+        width: var(--collectionBrowserCellWidth, 180px);
+      }
+
+      .inner-holder {
+        width: calc(var(--collectionBrowserCellWidth) - 10px);
+        height: 96%;
+        display: flex;
+        flex-direction: column;
+      }
+
+      .title-holder {
+        height: 40px;
+        margin-bottom: 5px;
+      }
+
+      .avatar-holder {
+        margin-bottom: 5px;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        max-height: 160px;
```

Comment:
Do you need this `max-height` since the avatar image has a fixed 160x160? I think if we changed the cell height, we'd want this section to grow to take up the available height.